src/common/runner/runner.go

```diff
@@ -72,11 +73,13 @@ func (r *Runner) Init(commands *clioptions.TagOptions) error {
 		}
 		switch p {
 		case "Terraform":
-			r.parsers = append(r.parsers, &tfStructure.TerraformParser{})
+			r.parsers = append(r.parsers, &tfStructure.TerrraformParser{})
```

Comment:
@sshintaku Please revert... Must have been a conflict I guess

Reply:
This has been reverted back.